src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -126,6 +126,9 @@ private boolean isDefault( final String attributeValue ) {
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
+		if ( AttributesProvider.getInstance().getSizes().contains( attribute ) && attributeValue.equals( "0px" ) ) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=32&issues=AWSLWuHMsk71k7lwmQ83&open=AWSLWuHMsk71k7lwmQ83)

[ISSUE_KEY]: AWSLWuHMsk71k7lwmQ83